luigi/contrib/gcs.py

```diff
@@ -249,6 +249,28 @@ def put(self, filename, dest_path, mimetype=None, chunksize=None):
 
         self._do_put(media, dest_path)
 
+    def _put_star(self, kwargs):
+        return self.put(**kwargs)
+
+    def put_multiple(self, filenames, directory_path, mimetype=None, chunksize=None, num_process=1):
+        args = [
+            {
+                'filename': filename,
+                'dest_path': os.path.join(directory_path, os.path.basename(filename)),
+                'mimetype': mimetype,
+                'chunksize': chunksize,
+            }
+            for filename in filenames
```

Comment:
There is no guard against submitting `filenames` as a string. The iterable won't fail until L245